### ISSUE TITLE
Install failed

### ISSUE THREAD

**Issue by Redpillbug** [fcitx/fcitx5-hangul#1]
I created and entered build directory inside "fcitx5-hangul-master" directory as said in INSTALL file, than executed

cmake ..
 but than I got:

CMake Error at CMakeLists.txt:5 (find_package):
  By not providing "FindECM.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ECM", but
  CMake did not find one.

  Could not find a package configuration file provided by "ECM" with any of
  the following names:

    ECMConfig.cmake
    ecm-config.cmake

  Add the installation prefix of "ECM" to CMAKE_PREFIX_PATH or set "ECM_DIR"
  to a directory containing one of the above files.  If "ECM" provides a
  separate development package or SDK, be sure it has been installed.

-- Configuring incomplete, errors occurred!
See also "/home/rpg/program-src/fcitx5-hangul-master/build/CMakeFiles/CMakeOutput.log".

I'm never used Cmake before, So I'm very confused. Any solutions? Thanks.
(currently my fcitx-hangul installed has some weird bug that eats all my RAM, making it unusable, So I had to type this in English.)

**Comment by wengxt** [comment]
You need this package: https://pkgs.org/download/extra-cmake-modules (check you distribution).

I'm more concerned about the issue you mentioned with fcitx-hangul, do you have any additional information? (e.g. does remove ~/.config/fcitx helps? or do you have any key sequence that can stably triggers the bug?)

**Comment by Redpillbug** [comment]
Hi, thanks for your reply, But Now I will close this issue since my problem is now solved.
I found a new input method that works nicely (also more easy to install, since it provides pre-compiled binary that can be installed via apt) Called [Nimf/님프](https://github.com/hamonikr/nimf) So I'm switching Nimf input method instead Fcitx :)

However, It seems like your solutions are also correct, since the command 
sudo apt-get install extra-cmake-modules
has teached me LMDE doesn't have that package pre-installed, and removing home/.config/fcitx did fix the weird RAM bug that was keep happening every time I booted my computer. (It started randomly, I'm not sure what kind of key sequence might have caused it.)
However, in cause you want to inspect the cause of this bug I will upload my ~/.config/fcitx file here, but to be honest, the only human understandable log(ficitx/log/crash.log)  just says: 
fcitx: BadWindow (invalid Window parameter)
So I believe it will be hard to find out why this happened.
Again, thanks you for your support.
[fcitx.zip](https://github.com/fcitx/fcitx5-hangul/files/5994684/fcitx.zip)
history.dat under the clipboard directory has been deleted since privacy reasons, and also that file alone was tediously large.
